Tweeter/render.js

```diff
@@ -0,0 +1,36 @@
+
+
+
+const Renderer = function () {
+
+    const renderPost = function (posts) {
+
+        $("#posts").empty()
+
+        let postText = ""
```

Comment:
you do not need to pre-define these variables. Define them when you use them in the for loop. It makes your code shorter and cleaner. 

also they can be `const` variables